# A PRAC action core with a name and a list of action roles 

string action_core_name

ActionRole[] action_roles
